### PR TITLE
[native]Add system config for global memory arbitration in velox

### PR DESCRIPTION
**Pull request by xiaoxmeng**
```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-native-execution/presto_cpp/main/common/Configs.cpp [hunk 1]
```diff
@@ -478,6 +479,10 @@ std::string SystemConfig::memoryArbitratorKind() const {
   return optionalProperty<std::string>(kMemoryArbitratorKind).value_or("");
 }
 
+bool SystemConfig::memoryArbitratorGlobalArbitrationEnabled() const {
+  return optionalProperty<bool>(kMemoryArbitratorGlobalArbitrationEnabled).value_or("");
```
Comment (tanjialiang) [inline presto-native-execution/presto_cpp/main/common/Configs.cpp:483]: .value_or(false)

**Review by tanjialiang — DISMISSED** [review]

File: presto-native-execution/presto_cpp/main/common/Configs.cpp (hunk 1 above)
Comment (xiaoxmeng) [reply presto-native-execution/presto_cpp/main/common/Configs.cpp:483]: All the system configs in Prestissimo doesn't provide the default here. I follow the same pattern.

**Comment by aditi-pandit** [comment]
@xiaoxmeng : Please can you add a release note documenting this property.

To clarify by default a query can reclaim from another query, but this property restricts arbitration to only look at other tasks of the same query ? Or is it something else ?

**Comment by xiaoxmeng** [comment]
> @xiaoxmeng : Please can you add a release note documenting this property.
> 
> To clarify by default a query can reclaim from another query, but this property restricts arbitration to only look at other tasks of the same query ? Or is it something else ?

By default, this option is off at Velox side. This provides a system config to turn it on in Prestissimo

**Review by tanjialiang — APPROVED** [review]

File: presto-native-execution/presto_cpp/main/common/Configs.cpp (hunk 1 above)
Comment (tanjialiang) [reply presto-native-execution/presto_cpp/main/common/Configs.cpp]: Yeah but you are assigning a string to a bool

**Review by tanjialiang — APPROVED** [review]

**Pull request merged**